docs/en/observability/cloud-monitoring/aws/monitor-aws-cloudtrail-firehose.asciidoc

```diff
@@ -0,0 +1,207 @@
+[[monitor-aws-cloudtrail-firehose]]
+= Monitor CloudTrail logs
+
+++++
+<titleabbrev>Monitor CloudTrail logs</titleabbrev>
+++++
+
+In this section, you'll learn how to monitor and analyze the CloudTrail logs you send to Elastic with Amazon Data Firehose. You will go through the following steps:
+
+- Install AWS integration in {kib}
+- Export Cloudtrail events to CloudWatch
+- Set up a Firehose delivery stream
+- Set up a subscription filter to route Cloudtrail events to a delivery stream
+- Visualize your CloudTrail logs in {kib}
```

Comment:
Should this be an ordered list instead?

Reply:
This list is more an overview / introduction of what follows. 
I would leave the bullets, also because this format is consistent with other 6 tutorials.